parentchain/src/weights.rs

```diff
@@ -0,0 +1,13 @@
+pub use frame_support::weights::{constants::RocksDbWeight, Weight};
+
+/// Weight functions needed for pallet_parentchain.
+pub trait WeightInfo {
+	fn set_block() -> Weight;
+}
+
+/// Weights for pallet_parentchain using the Integritee parachain node and recommended hardware.
+impl WeightInfo for () {
+	fn set_block() -> Weight {
+		10_000 as Weight
+	}
+}
```

Comment:
Where did you get these weights?

Reply:
This was decided by @gaudenzkessler and @brenzi